docs/base-maps.md

```diff
@@ -0,0 +1,29 @@
+# Base Maps
```

Comment:
This is basically replacing the current /docs/get-started/using-with-maps.md

I would recommend not changing its path because it maps to the published URL (deck.gl/docs/...) which is already linked to from other doc pages and various issues/discussions.

Reply:
moved this content to /docs/get-started/using-with-maps.md to preserve the path, but kept it under Overview in toc